guides/common/modules/proc_configuring-compliance-policy-deployment-methods.adoc

```diff
@@ -0,0 +1,36 @@
+[id="configuring-compliance-policy-deployment-methods_{context}"]
+= Configuring Compliance Policy Deployment Methods
+
+Use the following procedure to configure {Project} for the method that you have selected to deploy compliance policies.
+
+.Procedure
+* For the Ansible deployment method:
+** Ensure that you import the `theforeman.foreman_scap_client` Ansible role.
+
++
+For more information, see {ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].
+
++
+Then you must assign the created policy and the `theforeman.foreman_scap_client` Ansible role to a host or host group.
+To trigger the deployment, you must run the Ansible role on the host or host group either manually or set up a recurring job using remote execution.
+* For the Puppet deployment method:
+ifdef::satellite[]
+. Enable Puppet.
+endif::[]
+. Ensure the Puppet agent is installed on hosts.
+. Ensure that you import the Puppet environment that contains the `foreman_scap_client` Puppet module.
+For more information, see {ManagingConfigurationsPuppetDocURL}[_{ManagingConfigurationsPuppetDocTitle}_].
+
++
+Then you must assign the created policy and the `foreman_scap_client*` Puppet classes to a host or host group.
+Puppet triggers the deployment on the next regular run or you can run Puppet manually.
+Note that Puppet runs every 30 minutes by default.
+
+* For the manual deployment method, no additional {Project} configuration is required.
+ifdef::satellite[]
++
+For information on manual deployment, see https://access.redhat.com/solutions/6389101[How to set up OpenSCAP Policies using Manual Deployment option] in the _Red{nbsp}Hat Knowledgebase_.
+endif::[]
+
+.Additional resources
+* xref:deploying-compliance-policies_{context}[]
```

Comment:
```suggestion
```

IMHO an xref does not qualify as an "additional resource". I'd just drop it here to be honest.